src/components/AboutMeSection/AboutMeSection.js

```diff
@@ -36,8 +37,15 @@ const AboutMeSection = () => {
     }
   `);
 
+  const animatedSectionWrapper = useRef(null);
+  const id = 'aboutMe';
+
+  useEffect(() => {
+    onScrollAnimate(animatedSectionWrapper, id);
```

Comment:
you should verify if ref exists - look, when you are assigning ref it's null. 
Remember - ref changes does not influence on render, so will be not provided in deps array. It would be nice if you read about refs and hooks in React docs.